From 7be8a2eb28f016874386c7bd8570640a768dad2a Mon Sep 17 00:00:00 2001
From: XiaoDong Huang <derrick.huang@rock-chips.com>
Date: Thu, 6 Dec 2018 18:47:54 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368: change default sleep mode to
 RKPM_SLP_ARMOFF

The power consumption is no visible difference between RKPM_SLP_ARMOFF_LOGPD
mode and RKPM_SLP_ARMOFF mode. But RKPM_SLP_ARMOFF_LOGPD mode will cost
more time in suspend/resume flow. So we'd better use RKPM_SLP_ARMOFF in
most products. Here, we change RKPM_SLP_ARMOFF_LOGPD to RKPM_SLP_ARMOFF
in dtsi, that's a good way to make most products to use RKPM_SLP_ARMOFF.

Change-Id: Iec541a6831022ded2dad4298fc470bec7bd978f6
Signed-off-by: XiaoDong Huang <derrick.huang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 4b3aafc40d36..60e2a00de4c8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -2404,7 +2404,7 @@
 		rockchip,sleep-debug-en = <0>;
 		rockchip,sleep-mode-config = <
 			(0
-			| RKPM_SLP_ARMOFF_LOGPD
+			| RKPM_SLP_ARMOFF
 			| RKPM_SLP_PMU_PLLS_PWRDN
 			| RKPM_SLP_PMU_PMUALIVE_32K
 			| RKPM_SLP_SFT_PLLS_DEEP
-- 
2.35.3

